### ISSUE TITLE
Use of sort alias breaks peek on Linux

### ISSUE THREAD

**Issue by fsackur** [oising/poke#12]
On Linux, the alias `sort` => `Sort-Object` is not created by default, and `peek` inadvertently calls GNU sort:

```
❯ Get-Command Register-ArgumentCompleter | peek | gm
/usr/bin/sort: invalid option -- 'q'                                                                                 ]
Try '/usr/bin/sort --help' for more information.
/usr/bin/sort: cannot read: name: No such file or directory
                                                                                                                        
   TypeName: Pokeable.System.Management.Automation.CmdletInfo#1881356f-4c91-42aa-b2bf-f9f380d21066                      

Name            Modifier MemberType Definition
----            -------- ---------- ----------
Equals          public   Method     bool Equals(System.Object obj)
GetHashCode     public   Method     int GetHashCode()
GetType         public   Method     type GetType()
ToString        public   Method*    string ToString()
__GetBaseObject -        Method*    System.Management.Automation.CmdletInfo __GetBaseObject()
__GetModuleInfo -        Method*    psmoduleinfo __GetModuleInfo()
```

Workaround: `Set-Alias sort Sort-Object`

The issue is at:

- https://github.com/oising/poke/blob/a0ca5d92b/Poke.psm1#L345
- https://github.com/oising/poke/blob/a0ca5d92b/Poke.psm1#L675
- https://github.com/oising/poke/blob/a0ca5d92b/Poke.psm1#L728

**Comment by fsackur** [comment]
PR 13 is a minimal fix. Would you like me to add a line to the readme for v1.1.1, if you're willing to publish it?

**Comment by oising** [comment]
Thank you! Published as 1.1.2.0

https://www.powershellgallery.com/packages/poke/1.1.2.0
